src/layout/BaseLayout/_base-layout.scss

```diff
@@ -61,6 +61,21 @@
     margin: 0 1rem;
     position: absolute;
   }
+
+  @include desktop {
+    height: 100vh;
+
+    > div {
+      display: flex;
+      flex-direction: column;
+      height: 100%;
+
+      .l-content {
+        flex: 1;
+        min-height: 0;
+      }
+    }
+  }
```

Comment:
```suggestion
```


Instead add the following after line [162](https://github.com/canonical/juju-dashboard/pull/1547/files#diff-fd8132c704f92ac7d42cd11b60ec0393b0d08e6655e2e72c8d24c99d21909572R162):

```css
  .l-navigation.is-pinned {
    @include desktop {
      position: sticky !important;
    }
  }
```

Reply:
Thanks for pointing out this change! It is definitely simpler and better than my approach, and scales well with different contents. I've added also `overflow-x: scroll` to `.l-main` in order for the user to be able to scroll horizontally.